pkg/models/temp_user.go

```diff
@@ -82,24 +82,44 @@ type GetTempUsersQuery struct {
 }
 
 type GetTempUserByCodeQuery struct {
-	Code string
-
+	Code   string
 	Result *TempUserDTO
 }
 
+type Time struct{ time.Time }
 type TempUserDTO struct {
-	Id             int64          `json:"id"`
-	OrgId          int64          `json:"orgId"`
-	Name           string         `json:"name"`
-	Email          string         `json:"email"`
-	Role           org.RoleType   `json:"role"`
-	InvitedByLogin string         `json:"invitedByLogin"`
-	InvitedByEmail string         `json:"invitedByEmail"`
-	InvitedByName  string         `json:"invitedByName"`
-	Code           string         `json:"code"`
-	Status         TempUserStatus `json:"status"`
-	Url            string         `json:"url"`
-	EmailSent      bool           `json:"emailSent"`
-	EmailSentOn    time.Time      `json:"emailSentOn"`
-	Created        time.Time      `json:"createdOn"`
+	Id             int64          `json:"id" db:"id"`
+	OrgId          int64          `json:"orgId" db:"org_id"`
+	Name           string         `json:"name" db:"name"`
+	Email          string         `json:"email" db:"email"`
+	Role           org.RoleType   `json:"role" db:"role"`
+	InvitedByLogin *string        `json:"invitedByLogin" db:"invited_by_login"`
```

Comment:
Question: Why did these InvitedBy* types change to pointers to strings? I don't hate it, but I am curious.

Reply:
like your curiosity, now the data structure has to be accurate since we removed xorm magics (all nullable object has to be a pointer or sql.NullStringish)

Reply:
Capturing what we discussed already for posterity: we're going to change this so these are strings instead of pointers to ensure backwards compatibility. The change made sense on its own, but we don't want to risk callers running into unexpected `nil`s. We can revisit it after the xorm assassination is complete, if we feel a need!

Reply:
sounds good! ❤️